src/index.js

```diff
@@ -9,19 +9,29 @@ import PackageShow from './routes/package/package-show';
 
 export default class EHoldings extends Component {
   static propTypes = {
+    stripes: PropTypes.shape({
+      connect: PropTypes.func.isRequired
+    }).isRequired,
     match: PropTypes.shape({
       path: PropTypes.string.isRequired
     }).isRequired
   }
 
+  constructor(props) {
+    super(props);
+    this.ConnectedApp = props.stripes.connect(App);
+    this.ConnectedVendorShow = props.stripes.connect(VendorShow);
+    this.ConnectedPackageShow = props.stripes.connect(PackageShow);
```

Comment:
while you could argue that `ConnectedSomething` is clearer semantically, i think you can get away with `this.App = props.stripes.connect(App)`.  it should still be pretty clear what's going on and it looks a little bit cleaner.